autogen.pl

```diff
@@ -776,6 +776,43 @@ sub find_and_delete {
 # Find a specific executable and ensure that it is a recent enough
 # version.
 
+sub check_m4_gnu_extensions {
+    verbose "Checking whether m4 supports the gnu extensions\n";
+    my $m4 = $ENV{'M4'};
+    $m4 = "m4" unless (defined $m4);
+    my $result = `$m4 -g config/check_m4_gnu_extension.m4`;
```

Comment:
Don't you need to direct stderr to `/dev/null` (or better yet, `config.log`, or even better yet, use `OPAL_LOG_COMMAND` -- see `opal_functions.m4`)?  Otherwise, if something goes wrong, we'll see spurrious output in configure stdout / you can't see what happened by examining `config.log`.